spec/main.spec.js

```diff
@@ -941,6 +941,110 @@ describe('Honeybadger', function() {
     });
   });
 
+  describe('window.onunhandledrejection callback', function() {
+    describe('default behavior', function() {
+      beforeEach(function() {
+        if (typeof PromiseRejectionEvent === 'undefined') {
```

Comment:
`PromiseRejectionEvent` is unsupported in Edge and FireFox, apparently:

https://developer.mozilla.org/en-US/docs/Web/API/PromiseRejectionEvent

I added this filter to just skip the specs for these browsers. I didn't add anything to skip the event handler from being attached, but I don't *think* that's a problem, since it just won't be called in those browsers anyway?

Reply:
Yeah that seems to be the case. Tested it in Firefox locally and it Firefox just ignored the `window.onunhandledrejection ` function

Reply:
Also good catch! I didn't even realize this was not yet supported in all browsers. A little surprised Firefox is lagging behind on this one!